types/indexer.go

```diff
@@ -273,6 +298,14 @@ func prefixKeyForSigner(signer Address) []byte {
 	))
 }
 
+func prefixKeyForSignerAndHeight(signer Address, height int64) []byte {
+	return []byte(fmt.Sprintf("%s/%s/%s",
+		TxSignerKey,
+		signer,
+		elenEncoder.EncodeInt(int(height)), // totally safe right?
```

Comment:
`// totally safe right?` -> Do you think we'll hit an overflow issue or something? If so, we can just force the user to pass an int into this function.

Reply:
heh i think this was more of a joke to myself, I don't think we will be still babysitting v0 after 2^63-1 (or 2^31-1 on 32bit systems) blocks.

Reply:
💯

Reply:
Let's remove the joke with a real comment (so future readers aren't confused), but otherwise lgtm